src/FontInterface.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace HelloFresh\Ausraster;
+
+interface FontInterface
```

Comment:
Agreed. I need to put some more thought into the way the font works, because it's going to affect things other than Excel (word docs, pdf, whatever).